packages/storybook/stories/PowerPoint Live.stories.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  id="ppt-live"
+  title="Concepts/PowerPoint Live"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# PowerPoint Live
+
+Azure Communication Services (ACS) integrate PowerPoint Live into Teams meetings, enhancing the presentation experience. This feature allows real-time engagement with presentations, making it an invaluable tool for various scenarios.
+
+## Overview
+
+In the ACS UI library, users participate as attendees to view PowerPoint Live sessions. The interaction capabilities for attendees are set and controlled by the lead presenter of the session.
+
+For a detailed breakdown of the capabilities available to each role during a PowerPoint Live session, please refer to the [PowerPoint Live documentation](https://support.microsoft.com/en-us/office/present-from-powerpoint-live-in-microsoft-teams-28b20e74-7165-499c-9bd4-0ad975d448ad). For more technical details on how to implement these features using the web calling SDK, consult the [Calling SDK documentation](https://learn.microsoft.com/en-us/azure/communication-services/concepts/voice-video-calling/calling-sdk-features).
+
+PowerPoint Live is ideally suited for:
+
+- Corporate presentations
+- Educational lectures
+- Remote training sessions
+- Webinars
+
+![pptlive example](images/pptlive-example.png 'Left: Teams app, Right: ACS UI library')
+
+<sub>
+  <sup>Figure: PowerPoint Live interface comparison. Left: Teams app. Right: ACS UI library.</sup>
+</sub>
+
+## Implementing with the VideoGallery Components
+
+The VideoGallery component is used to display PowerPoint Live sessions. This multi-functional component also supports video streams and screen sharing. Here’s a brief guide to integrating it:
+
+```typescript
+// Import the necessary dependencies
+import { VideoGallery } from '@azure/communication-react';
+
+// Use the VideoGallery component to display the presentation
+function PPTLiveComponent() {
+  // Definition of a mock object simulating a PowerPoint live stream.
+  const mockPptLiveStream = {
+    id: 1,
+    isAvailable: true,
+    renderElement: mockPptLiveElement as HTMLElement
```

Comment:
can we use call.features(pptLive).targetElement instead, looks like this code won't run anyways as we don't have a `mockPptLiveElement`